server/src/main/java/se/fortnox/reactivewizard/server/RxNettyServer.java

```diff
@@ -42,12 +42,12 @@ public RxNettyServer(ServerConfig config, CompositeRequestHandler compositeReque
     }
 
     private static HttpServer<ByteBuf, ByteBuf> createHttpServer(ServerConfig config) {
-        return HttpServer.newServer(config.getPort())
+        return config.isEnabled() ? HttpServer.newServer(config.getPort())
```

Comment:
This would be so much easier to follow with just a simple guard clause instead :)

Reply:
fixed!